src/schema/submission.yaml

```diff
@@ -1181,7 +1161,7 @@ slots:
       The members associated with a committee.
     range: Member
     multivalued: True
-    inlined: true
+    inlined: false
```

Comment:
I think we should only use inlined in slot_usage but not in slots.

Reply:
Fixed.